File: extensions/panache/hibernate-reactive-panache-kotlin/deployment/src/test/java/io/quarkus/hibernate/reactive/panache/test/PanacheEntityMetaModelTest.java
package io.quarkus.hibernate.reactive.panache.test;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import io.quarkus.hibernate.reactive.panache.PanacheEntity_;

public class PanacheEntityMetaModelTest {
    @Test
    public void testMetaModelExistence() {
        Assertions.assertEquals("id", PanacheEntity_.ID);
    }
}

File: extensions/panache/hibernate-reactive-panache-kotlin/deployment/src/test/kotlin/io/quarkus/hibernate/reactive/panache/kotlin/test/PanacheEntityMetaModelTest.kt
package io.quarkus.hibernate.reactive.panache.kotlin.test

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheEntity_
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PanacheEntityMetaModelTest {
    @Test
    fun testMetaModelExistence() {
        Assertions.assertEquals("id", PanacheEntity_.ID)
    }
}